agreements/project-contribution/terms-and-conditions.md

```diff
@@ -0,0 +1,292 @@
+Details about fee structures and any adjustments are available upon request.
+
+#### Platform
+
+The Sponsor currently uses the Open Collective platform (the “Platform”) to manage restricted funds for hosted projects.
+Upon execution of the Fiscal Sponsorship Agreement, a new Collective (or a Collective Project) will be created for the restricted fund.
+
+The Sponsor may switch to another platform with similar capabilities if necessary, with advance notice to the Project.
+
+#### Expenses and Disbursements
+
+Projects must submit expense reports (with receipts and documentation) through the Platform.
+
+All expenses must be approved by the Sponsor to ensure they are valid and in alignment with the Sponsor's tax-exempt purposes under the terms of this agreement before disbursement.
+
+Upon approval by the Sponsor, the Sponsor will reimburse those expenses (from the Project's restricted fund) through the Platform using a mutually agreed payment option.
+
+The Project must work with the Sponsor to ensure financial records are accurate, including management/reconciliation of funds received or expenses paid using mechanisms other than the Platform.
+
+### Transparency and Reporting
+
+Projects must maintain transparency in their financial activities for funds managed under the Sponsor’s oversight and provide regular updates to the Sponsor.
+The Sponsor will compile an annual financial report for each Project, detailing income, expenses, and remaining balances for these Sponsor-managed funds.
+
+Projects will be asked to review and verify the financial summary provided by the Sponsor to ensure accuracy.
+Any discrepancies should be reported to the Sponsor within 30 days of receiving the report.
+
+Projects are encouraged to keep their expense tracking up to date on the Open Collective platform for real-time transparency.
+
+### Independent Financial Management
+
+Individuals, companies, or groups may raise and manage funds independently in support of their contributions to the Project.
+These funds are not part of the fiscal sponsorship arrangement and must remain separate from the Sponsor’s financial management systems.
+Those managing independent funds are solely responsible for compliance with all relevant laws and regulations.
+
+While alignment with the Sponsor’s policies is encouraged where possible, these independent activities must remain separate from the Project’s operations under fiscal sponsorship.
+
+## Continuity and Leadership Responsibilities
+
+Project leadership and operations naturally evolve due to changes in roles and responsibilities, or to unforeseen circumstances.
+
+### Succession Planning and Leadership Transition
+
+In this section, **“Successor”** refers to a qualified individual who assumes the role of Project Leader or Maintainer.
+As specified in the [Fiscal Sponsorship Agreement][fs-landing], a Successor must agree to the terms of that Agreement and become a Signatory.
```

Comment:
Should this refer to "Future Project Leader" instead of "Successor"?

To me it's confusing to use the term "Successor", which in the Fiscal Sponsorship Agreement refers to non-profits that may take over a project.

As this is talking about replacing a project leader, or the like, as defined in Recitals of the FSA, it should be clear they're separate

Reply:
Let me try that. I agree on reuse of Successor (which is why I redefined it here), but I also think the meaning of the word (the leader that will take your place when you leave), is useful here. Future project leader is kind of a mouthful to repeat throughout the section.